.github/workflows/deploy-infra.yaml

```diff
@@ -18,7 +18,8 @@ on:
         - staging and production
         default: staging
 env:
-  region: 'eu-west-1'
+  REGION: 'eu-west-1'
+  RELEASES_REPOSITORY: hms-dbmi-cellenics/releases
```

Comment:
you can potentially be able to replace `hms-dbmi-cellenics` with `${{ github.repository_owner }}` and remove the org hardcoding. Can you try and see if it works in here? We use it in other workflows, but not under `env`.

Reply:
can you push the change?